package.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "filmy-ts",
   "version": "0.1.0",
-  "private": true,
+  "private": false,
   "main": "src/index.tsx",
   "homepage": "https://awnigharbia.github.io/filmy/",
+  "engines": {
+    "node": ">12"
+  },
```

Comment:
You also need to add this file `.npmrc` in the root of your repo to make it really work. https://github.com/john-d-pelingo/vite-boilerplate/blob/master/.npmrc

Reply:
Okay I did it!